schemas/2017-05-10/Microsoft.Resources.json

```diff
@@ -7,27 +7,25 @@
     "deployments": {
       "type": "object",
       "properties": {
-        "type": {
-          "type": "string",
-          "enum": [
-            "Microsoft.Resources/deployments"
-          ]
-        },
         "apiVersion": {
           "type": "string",
           "enum": [
             "2017-05-10"
           ]
         },
-        "subscriptionId": {
-          "type": "string",
-          "description": "The subscription to deploy to"
-        },
-        "resourceGroup": {
```

Comment:
>resourceGroup [](start = 9, length = 13)

seems the "resourceGroup" and "subscriptionId" properties are being removed. we may want to keep them.. they are (only) supported for nested deployment.

Reply:
Again looks like an inaccuracy in swagger - should these be added to the following?
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/resources/resource-manager/Microsoft.Resources/stable/2017-05-10/resources.json#L2285

Reply:
this is a tricky part too. these two properties only work in template resources. they are not supported in PUT so swagger doesn't contain them.